### PR TITLE
zaproxy/zaproxy #1937.  This is a request for review to move Format String from Beta to Release

### PR DESCRIPTION
**Pull request by msrader**
I want to move Format String from Beta to Release.  I also updated ascanrules.html to
re-alphabetize the listings.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrules/ZapAddOn.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<url/>
 	<changes>
 	<![CDATA[
-        Change Path Traversal scanner to also check HTML responses in decoded form.
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrules/ZapAddOn.xml]: This change should be kept, the add-on was not yet released.

**Comment by msrader** [comment]
All Fixed.

**Comment by thc202** [comment]
Thanks!
Will you open a PR to remove `FormatString` from beta?

**Comment by msrader** [comment]
Surely.  I'll get it done tonight.

**Comment by thc202** [comment]
OK.

**Comment by msrader** [comment]
Done

**Pull request merged**